examples/networklayer_udp.c

```diff
@@ -71,41 +68,44 @@ typedef struct {
 } ServerNetworkLayerUDP;
 
 /** Accesses only the sockfd in the handle. Can be run from parallel threads. */
-static void writeCallbackUDP(UDPConnection *handle, UA_ByteStringArray gather_buf) {
-	UA_UInt32 total_len = 0, nWritten = 0;
-	struct iovec iov[gather_buf.stringsSize];
-	for(UA_UInt32 i=0;i<gather_buf.stringsSize;i++) {
```

Comment:
I am new to the codebase so was following history in a number of these changes. Happy for any insight you can provide.

In this case gather_buf  is a UA_ByteStringArray which is no longer in the codebase. When it was it was an array of UA_ByteString with a length of stringsSize, seeing as we can now only pass a single UA_ByteString to the method the for-loop seemed unnecessary.